lib/context.js

```diff
@@ -76,7 +76,17 @@ class Context {
    * // Load config from .github/myapp.yml in the repository
    * const config = await context.config('myapp.yml');
    *
-   * if(config.close) {
+   * if (config.close) {
+   *   context.github.issues.comment(context.issue({body: config.comment}));
+   *   context.github.issues.edit(context.issue({state: 'closed'}));
+   * }
+   *
+   * @example <caption>Using a <code>defaultConfig</code> object.</caption>
+   *
+   * // Load config from .github/myapp.yml in the repository and combine with default config
+   * const config = await context.config('myapp.yml', {comment: 'Make sure to check all the specs.'});
```

Comment:
nit: `;` is ❌ for standard

Also weird there are other places in this file not standard linted :thinking:

Reply:
@hiimbex Good catch 👌 I think the reason we're seeing these not get linted is that the example code is actually in a comment instead of just being in the file itself.

Reply:
Is there an eslint plugin to check jsdoc comments similar to https://github.com/probot/probot/pull/252?

Reply:
@bkeepers a quick search seems to show plugins for linting JSDoc comments, but not specifically code in JSDoc examples.

I'll remove the `;` in the snippets relevant to this PR, but it should obviously be consistent everywhere. Separate PR or this one?

Reply:
I'm having a vision of additional functionality for probot/linter lol. I'd go with separate 👍